UnityClient/Assets/Scripts/TowerDefence/Enemy.cs

```diff
@@ -5,10 +5,14 @@
 
 public class Enemy : MonoBehaviour
 {
+    public float startSpeed = 10f;
+
+    [HideInInspector] public float speed;
+    
     public float startHealth = 100;
     private float health;
 
-    public int worth = 50;
+    //public int worth = 50;
 
     public GameObject deathEffect;
```

Comment:
public fields are a cardinal sin and go against the convention of the codebase. If you need Unity inspector support, please convert this to a `SerializeFieldAttribute` private field, as per the other places in the codebase. If you need access to this value outside of this type, please use a property instead.

If it does become private, please follow the codebase's private field convention and define the name `_inThisFormat`, as per other places in the codebase.